app/templates/components/profile-nav.hbs

```diff
@@ -81,6 +81,12 @@
             {{#link-to 'organization.repositories' model }}Repositories{{/link-to}}
           </li>
 
+          {{#if isOrganizationAdmin}}
```

Comment:
I tried the org PR deployment and this tab shows even though no actual settings are present, maybe this needs a `proVersion` check too?